react/features/poll/components/PollsList/web/CreatePollButton.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import React, { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { openPollCreationPage } from '../../../actions';
+import {
+    FooterButton
+} from '../../PollPanel/styled';
+
+export function CreatePollButton() {
+    const dispatch = useDispatch();
+    const { t } = useTranslation();
+    const showPollList = useSelector(state => state['features/poll'].showPollList);
+    const pollPaneMode = useSelector(state => state['features/poll'].pollPaneMode);
+    const createPoll = useCallback(() => {
+        console.log('CREATE NEW POLL');
+        dispatch(openPollCreationPage(), [ dispatch ]);
+    });
+
+    if (pollPaneMode !== 'PollsList') {
```

Comment:
As mentioned above, we can just have the parent component determine if we should render this component.